### PR TITLE
create vercel lambdas

### PR DESCRIPTION
**Pull request by tiirent**

### CODE REVIEW

File: api/hello.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export default (req, res) => {
```
Comment (josephli036) [inline api/hello.js:1]: Don't forget to delete this file

File: api/item.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import client from "../src/services/poe-client.js";
+
+export default (req, res) => {
+  const headers = {
```
Comment (josephli036) [inline api/item.js:4]: Can DRY this to a utils file, since we use it everywhere

File: src/store/store-manager.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  poll() {
+    console.log("polledd...");
+    console.log(JSON.stringify(store.getters.feed));
+    // setTimeout(() => this.poll(), 1000)
+    return;
+  },
+
+  // checkItem() queries item and updates itemList, returning list of resultIds
+  checkItem(itemId) {
+    let item = store.getters.trackedItems[itemId];
```
Comment (josephli036) [inline src/store/store-manager.js:15]: Oh wait, so ur gonna keep track of stored items?

File: src/services/api-client.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import { API } from "./axios-config.js";
+
+export default {
+  async getQuery(id) {
```
Comment (josephli036) [inline src/services/api-client.js:4]: 👍  nice, i'll start using these

File: src/store/store-manager.js (hunk 3 above)
Comment (tiirent) [reply src/store/store-manager.js:15]: this is you

**Pull request merged**